charts/jenkins-x-crds/templates/sourcerepositories-crd.yaml

```diff
@@ -5,299 +5,69 @@ apiVersion: apiextensions.k8s.io/v1
-    name: Description
-    type: string
-  conversion:
-    strategy: None
   group: jenkins.io
   names:
-    categories:
-    - all
     kind: SourceRepository
     listKind: SourceRepositoryList
     plural: sourcerepositories
-    shortNames:
-    - sourcerepo
-    - srcrepo
-    - sr
     singular: sourcerepository
   scope: Namespaced
-  validation:
-    openAPIV3Schema:
-      description: SourceRepository is the metadata for an Application/Project/SourceRepository
-      properties:
-        apiVersion:
-          description: 'APIVersion defines the versioned schema of this representation
-            of an object. Servers should convert recognized schemas to the latest
-            internal value, and may reject unrecognized values. More info: https://git.k8s.io/community/contributors/devel/api-conventions.md#resources'
-          type: string
-        kind:
-          description: 'Kind is a string value representing the REST resource this
-            object represents. Servers may infer this from the endpoint the client
-            submits requests to. Cannot be updated. In CamelCase. More info: https://git.k8s.io/community/contributors/devel/api-conventions.md#types-kinds'
-          type: string
-        metadata:
-          description: ObjectMeta is metadata that all persisted resources must have,
-            which includes all objects users must create.
-          properties:
-            annotations:
-              description: 'Annotations is an unstructured key value map stored with
-                a resource that may be set by external tools to store and retrieve
-                arbitrary metadata. They are not queryable and should be preserved
-                when modifying objects. More info: http://kubernetes.io/docs/user-guide/annotations'
-              type: object
-            clusterName:
-              description: The name of the cluster which the object belongs to. This
-                is used to distinguish resources with same name and namespace in different
-                clusters. This field is not set anywhere right now and apiserver is
-                going to ignore it if set in create or update request.
-              type: string
-            creationTimestamp:
-              description: Time is a wrapper around time.Time which supports correct
-                marshaling to YAML and JSON.  Wrappers are provided for many of the
-                factory methods that the time package offers.
-              format: date-time
-              type: string
-            deletionGracePeriodSeconds:
-              description: Number of seconds allowed for this object to gracefully
-                terminate before it will be removed from the system. Only set when
-                deletionTimestamp is also set. May only be shortened. Read-only.
-              format: int64
-              type: integer
-            deletionTimestamp:
```

Comment:
Need to check why some of these fields are missing in the generated crd manifests :thinking: